README.md

```diff
@@ -1,10 +1,10 @@
-This action allow you to build your JS file and get only compiled files (run it on node:10:16)
+This action allow you to install and build your JS file and get only compiled files (run it on node:10:16, 10.x, 12.x, 12.14.1)
```

Comment:
As I said in my previous comment, keep it simple. That's nice of you to make several tags available, but you're not going to come back in this repository every time a new patch version of node is released. You could provide instead:
* 10
* 12
* latest

Reply:
Problem is we already use the 10.16 so if we want to moove all modules on github we'll probably need it.
And yes it's a good time to try to update, but since it means to test an entire module, i think it will just delay again the migration ...

Reply:
I have been looking at the way the nodejs and npm teams maintain their repository and they're sensible to the semantic versionning, which means updating to the newer version on the same major version should not break the build nor the way our modules work.

I understand the risk you're talking about though, and remain opened for adding minor & patch versions later if it appears we cannot build anymore with earlier versions of node.

On the other side any security issue would not be fixed if we never upgrade (BTW there is a new version to be released tomorrow: https://nodejs.org/en/blog/vulnerability/february-2020-security-releases/). That's why I'm in favor of limiting the tags we maintain in this action to the major ones.

Reply:
yes the problem does not come from NodeJS but from the dependencies, i just know that 10.16 build might not work on 10.18 because dependencies can actually change :/ (but this was on Node 7 or 8 let's hope it don't exist anymore).

Do you still want to remove the 10.16 tag ? i'm okay to remove 12.14.1 tag and keep only 12 but if we do the same for the 10 just know that we will delay the migration to github again.

Reply:
If that comes from the dependencies, this could be related to the versions allowed in the `package.json` file. Let keep the major versions only.

Reply:
I agree, I've made the changes to just the 10,12 and latest versions.